demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -886,55 +871,55 @@ static BaseType_t prvBackoffForRetry( BackoffAlgorithmContext_t * pxRetryParams
  * PAL implementation for your platform in aws_ota_pal.c to see what it
  * does for you.
  *
- * @param[in] event Event from OTA lib of type OtaJobEvent_t.
+ * @param[in] xEvent Event from OTA lib of type OtaJobEvent_t.
  * @return None.
  */
-static void otaAppCallback( OtaJobEvent_t event,
-                            const void * pData );
+static void prvOtaAppCallback( OtaJobEvent_t xEvent,
+                               const void * pData );
 
 
 /**
  * @brief Common callback registered with MQTT agent to receive all publish packets.
  * Packets received using the callback is distributed to subscribed topics using subscription manager.
  *
- * @param[in] pMqttAgentContext MQTT agent context for the connection.
- * @param[in] packetId Packet identifier for the packet.
- * @param[in] pPublishInfo MQTT packet information which stores details of the
+ * @param[in] pxMqttAgentContext MQTT agent context for the connection.
+ * @param[in] usPacketId Packet identifier for the packet.
+ * @param[in] pxPublishInfo MQTT packet information which stores details of the
  * job document.
  */
-static void prvIncomingPublishCallback( MQTTAgentContext_t * pMqttAgentContext,
-                                        uint16_t packetId,
+static void prvIncomingPublishCallback( MQTTAgentContext_t * pxMqttAgentContext,
+                                        uint16_t usPacketId,
                                         MQTTPublishInfo_t * pxPublishInfo );
```

Comment:
Function lines are always written in a single line. Please apply to the rest as well.

Reply:
The statement states function names should be in a single line